rcv/nes56_cv_manager.py

```diff
@@ -83,8 +85,10 @@ def connect_to_roborio(self):
                 self._client_socket = None
```

Comment:
It is not a must, but for clearness and consistency you should set here the self._connected to false